app/src/main/java/me/ferdz/tutorfinder/models/UserSignupRequest.java

```diff
@@ -0,0 +1,62 @@
+package me.ferdz.tutorfinder.models;
+
+
+/**
+ * Basic signup request for both a student or a tutor
+ */
+public class UserSignupRequest {
+    private String firstName;
+    private String lastName;
+    private String email;
+    private String password;
+    private String confirmPassword;
+    private boolean isTutor;
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public void setFirstName(String firstName) {
+        this.firstName = firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String email) {
+        this.email = email;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+    public String getConfirmPassword() {
+        return confirmPassword;
+    }
+
+    public void setConfirmPassword(String confirmPassword) {
+        this.confirmPassword = confirmPassword;
+    }
+
+    public boolean isTutor() {
+        return isTutor;
+    }
+
+    public void setTutor(boolean tutor) {
+        isTutor = tutor;
```

Comment:
Why not `this.isTutor` 👀

Reply:
The auto-generation for fields did that lmao, I'll change

Reply:
👀